dali/test/python/test_external_source_dali.py

```diff
@@ -204,6 +204,20 @@ def test_dtype_arg():
     src_pipe.run()
 
 
+def test_dtype_arg_multioutput():
+    batch_size = 2
+    src_data = [
+        [[np.ones((120, 120, 3), dtype=np.uint8)]*batch_size,
+         [np.ones((120, 120, 3), dtype=np.float32)]*batch_size]
+    ]
+    src_pipe = Pipeline(batch_size, 1, 0)
+    src_ext, src_ext2 = fn.external_source(source=src_data, device='cpu', num_outputs=2,
+                                           dtype=[DALIDataType.UINT8, DALIDataType.FLOAT])
+    src_pipe.set_outputs(src_ext, src_ext2)
+    src_pipe.build()
+    src_pipe.run()
```

Comment:
Can you validate output (data and types)?

Reply:
Done